db/fixtures/miq_report_formats.yml

```diff
@@ -1,7 +1,10 @@
 ---
 :defaults_and_overrides:
   :formats_by_path:
+    :ContainerVolume-capacity: null
+    :ContainerVolumeKubernetes-capacity: null
```

Comment:
just FYI , ContainerVolumeKubernetes subclass is completely unused.
(I tried to drop it, #15646, we concluded I should also do a migration but never got around to it)